modules/sno-du-applying-the-extra-installation-manifests.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: CONCEPT
+[id="sno-du-applying-the-extra-installation-manifests_{context}"]
+= Applying the extra installation manifests
+
+To apply the distributed unit (DU) configuration to the single node cluster, the following
+extra installation manifests need to be included during installation:
+
+* Enable workload partitioning.
```

Comment:
We have already [enabled workload partitioning](https://github.com/openshift/openshift-docs/pull/44242/files#diff-cb01fda5b4538dddf7991b5835003e50b152eeb7f224c9cbf64f87fc8b476b5fR7). Do we have to do it again? Make sure it was done?

Reply:
This is an extra installation manifest located in the private repo.